File: src/main/java/com/ddd/trainings/proposal/domain/TrainingProposalFactory.java
package com.ddd.trainings.proposal.domain;

import com.ddd.trainings.idea.domain.IdeaId;

public class TrainingProposalFactory {

    public static TrainingProposal from(IdeaId ideaId) {
        return new TrainingProposal(ProposalId.create(), ideaId, null, "");
    }
}

File: src/Ddd/Trainings/Proposal/Domain/TrainingProposalFactory.cs
using Ddd.Trainings.Idea.Domain;

namespace Ddd.Trainings.Proposal.Domain
{
    public static class TrainingProposalFactory
    {
        public static TrainingProposal From(IdeaId ideaId)
        {
            return new TrainingProposal(ProposalId.Create(), ideaId, null, "");
        }
    }
} 